sidebarsDocs.js

```diff
@@ -291,7 +291,7 @@ module.exports = {
       label: 'Arduino',
       items: [
         'use-the-network/devices/development/arduino/mkr-wan-1310',
-        'use-the-network/devices/development/arduino/lora-vision-shield'
+        'use-the-network/devices/development/arduino/lora-vision-shield/lora-vision-shield'
```

Comment:
In addition to these sidebar changes, you'll also want to make a new section that mimics [this](https://github.com/helium/docs/blob/b1b57adcb6233331c4cee3e7bbce914a04a9698d/sidebarsDocs.js#L1178). This will show the side bar when you are on `/use-the-network/devices/development/arduino/lora-vision-shield/arduino/`, which it is currently lacking.

Reply:
Yes @kent-williams I am just getting my brain around where the name for this comes from

for example  ```helteccubecelhtccab01```  It looks like it has just be made up. I have been looking for a reference to the name. If I just make it up then I will be finished soon. 


```

  helteccubecelhtccab01: [
   {
      type: 'link',
      label: '<- HTCC-AB01',
      href: '/use-the-network/devices/development/heltec/cubecell-dev-board'
    },
    {
      type: 'category',
      label: 'Firmware',
      items: ['use-the-network/devices/development/heltec/cubecell-dev-board/arduino', 'use-the-network/devices/development/heltec/cubecell-dev-board/platformio'],
      collapsed: false,
    },
  ],

```

Reply:
@kent-williams  my attempt seems to break the sidebarsDocs.js file cann you have a look at it?

```
  loravisionshield: [
   {
      type: 'link',
      label: '<- Arduino LoRa Vision Shield',
      href: '/use-the-network/devices/development/arduino/lora-vision-shield/lora-vision-shield'
    },
    {
      type: 'category',
      label: 'Firmware',
      items: ['use-the-network/devices/development/arduino/lora-vision-shield/lora-vision-shield/arduino'],
      collapsed: false,
    },
  ],
```

Reply:
The name just needs to be unique for the file.  You can name it something like `loravisionshield`, and then update the rest of the fields appropriately for this doc.

Reply:
@kent-williams can you check why my addition to ```sidebarsDocs.js``` seems to break the code.

link [here](https://github.com/hpssjellis/docs-1/blob/a946345adb8ffa82892d6ae4604bded1e1cb3e48/sidebarsDocs.js#L1197-L1209)

Reply:
This is the build error:
```
 Error: Bad sidebars file.
 These sidebar document ids do not exist:
 - use-the-network/devices/development/arduino/lora-vision-shield
 - use-the-network/devices/development/arduino/lora-vision-shield/lora-vision-shield/arduino,
```
These are available, use these instead!
```
 - use-the-network/devices/development/arduino/lora-vision-shield/arduino
 - use-the-network/devices/development/arduino/lora-vision-shield/lora-vision-shield
```